pkg/common/peertracker/getpid.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _GETPID_H
+#define _GETPID_H
+
+#include <Windows.h>

```

Comment:
Nitpick: nothing in this header needs a definition from Windows.h. It can be included solely from getpid.c.